src/mrinufft/trajectories/tools.py

```diff
@@ -420,23 +420,34 @@ def get_gradient_times_to_travel(
 
     Returns
     -------
-    n_ramp_down: The timing values for the ramp down phase.
-    n_ramp_up: The timing values for the ramp up phase.
-    n_plateau: The timing values for the plateau phase.
-    gi: The intermediate gradient values for trapezoidal or triangular waveforms.
+    The timing values for the ramp down phase.
+    The timing values for the ramp up phase.
+    The timing values for the plateau phase.
+    The intermediate gradient values for trapezoidal or triangular waveforms.
```

Comment:
you can still names the return values, it helps understanding the code directly 

https://numpydoc.readthedocs.io/en/latest/format.html#returns